### PR TITLE
Adds a suicide message for the RPD

### PR DESCRIPTION
**Pull request by Lijah99**

### CODE REVIEW

File: code/game/objects/items/weapons/RPD.dm [hunk 1]
```diff
@@ -168,6 +168,12 @@ var/global/list/RPD_recipes=list(
 /obj/item/weapon/pipe_dispenser/attack_self(mob/user)
 	show_menu(user)
 
+/obj/item/weapon/pipe_dispenser/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] points the end of the RPD down \his throat and presses a button! It looks like \he's trying to commit suicide...</span>")
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/RPD.dm]: this should spawn a pipe where they are and gib them

Comment (Xhuis) [inline code/game/objects/items/weapons/RPD.dm]: but it's a _pipe_
why would they explode

Comment (Iamgoofball) [inline code/game/objects/items/weapons/RPD.dm]: compare the size of a disposal pipe to the size of a player

Comment (Xhuis) [inline code/game/objects/items/weapons/RPD.dm]: one would assume this is a normal atmos pipe

Comment (Iamgoofball) [inline code/game/objects/items/weapons/RPD.dm]: compare the length of an atmos pipe to a player

Comment (Xhuis) [inline code/game/objects/items/weapons/RPD.dm]: it still wouldn't make them gib

Comment (duncathan) [inline code/game/objects/items/weapons/RPD.dm]: An atmos pipe has a height of 1m and a radius of approximately 15cm
In freedom units that's a height of about 1 yard and a diameter of about 1 foot
You don't want that shit inside of you

Comment (Lijah99) [inline code/game/objects/items/weapons/RPD.dm]: Length isn't as important as width when it comes to ripping someone apart.

Comment (duncathan) [inline code/game/objects/items/weapons/RPD.dm]: A diameter of 30 cm would tear open your chest at LEAST

**Comment by Fox-McCloud** [comment]
Who care if it's not completely realistic--suicide_acts are meant to be melodramatic and hilarious--what's the big deal about it gibbing them? Suicide is already a mark of "I don't want to be in the round anymore" anyways.

Also, I'm pretty sure that spawning a pipe of that size would cause HORRENDOUS levels of physical trauma to the person beyond just "lol brute damage", so I wouldn't say it's completely unrealistic, either.

**Comment by duncathan** [comment]
Diameter of atmos pipe = ~12 inches
Average thickness of human chest = 9.5 inches (according to http://www.fas.harvard.edu/~loebinfo/loebinfo/Proportions/humanfigure.html)

**That shit will tear your chest apart.**

File: code/game/objects/items/weapons/RPD.dm (hunk 1 above)
Comment (Miauw62) [inline code/game/objects/items/weapons/RPD.dm]: 2lewd4me

**Comment by PKPenguin321** [comment]
\>arguing over whether or not we should make somebody explode when they kill themselves in a video game

**Comment by Cheridan** [comment]
serious business

**Comment by ExcessiveUseOfCobblestone** [comment]
Is this what ERPers have come to now, using Suicide messages as a vector to act lewd?

**Pull request merged**